torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -138,6 +138,12 @@ def is_public_torch_api(target):
     def is_safe_torch_function(target):
         """Allowlisted torch functions"""
         function_name = f"{target.__module__}.{target.__name__}"
+        # Allow torch.autograd.function.FunctionCtx if custom autograd functions are allowed
+        if function_name == "torch.autograd.function.FunctionCtx":
```

Comment:
I could have made this unconditionally a safe torch function and relied on line 163 to catch the corresponding autograd higher order op, but I think having it be handled separately is a bit more clear in terms of code reading